src/bp/ui-shared/src/telemetry/offlineServerFallback.ts

```diff
@@ -0,0 +1,49 @@
+import axios, { AxiosInstance } from 'axios'
```

Comment:
This is your main file and it's the only file in the folder. Name it "index.ts" instead.

Reply:
This pr is a fragmented one. At first, it was supposed to have more files and changes, but to make better incremental changes it was separated into multiple that are going to be added later. Each of these files his its own entry point.

Reply:
Still, name it index.ts and put all entry points here. If you happen to add other files afterwards, you can add their entry points here.

Reply:
Fixed in d31fd8f!